### ISSUE TITLE
Write a javascript app to list days-since-update for binderhubs

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/mybinder.org-deploy#945]
I think with a pretty small amount of javascript, we could do something like:

* Give us a link to a config file for your BinderHub
* The JS finds the hash used for the BinderHub (or r2d, or whatever) chart
* It compares this against the latest published chart
* It checks the dates of the commit hashes for these two values
* It displays a big **It's been XXX days since this BinderHub was updated** message

That's it.

Might be a nice way to keep us honest in updating BinderHub, and also to check how other BinderHubs are doing as we continue to support more of them. What do folks think?

**Comment by betatim** [comment]
This would be cool. A dashboard of sorts that shows what version each binderhub we know of is on.

https://github.com/jupyterhub/binderhub/pull/821 would help with getting the information about versions deployed to a hub.

One thing I think we should settle for for v1 is to show "Hub X is using version v123 which was released N days ago." Trying to work out when it was actually deployed is a bit trickier.

**Comment by choldgraf** [comment]
@betatim yep that sounds good to me as well, I don't imagine it being something too complex, just a little get-ing and YAML parsing and some big letters output :-)
